templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -231,16 +231,49 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
         </backend>
 
         <!--
-        <backend type="generic_s3" id="s3_netapp01" allow_selection="false" weight="0" store_by="uuid" name="Short term storage for method development">
-            <description>S3 based objetct storage matained by the compute center of the University of Freiburg.
-                         This storage is purged weekly and so it is only appropriate for short term methods development and such.
-                         The rapid deletion of stored data enables us to provide this storage without a quota.
-                         More information about our storage offerings can be found [here](https://galaxyproject.org/eu/).
+        
+        Q:
+        - use_reduced_redundancy - is this even available in Netapp? If so, should we use it for the scratch?
+        - max_chunk_size of 250 is that ok?
+        - `s3_netapp01_private` should this be `s3_netapp02_private` or should we use a separate bucket?
```

Comment:
A clean/new bucket would be nice.

Reply:
We need to talk to Stefan about that. He should have 2 tenants. And we can clean all of them up if needed.

ping @sj213

Reply:
I would also vote for separate buckets